### ISSUE TITLE
Can't run 0.0.20 from node_modules

### ISSUE THREAD

**Issue by mscharley** [cjoudrey/graphql-schema-linter#68]
I can't even get a version number out of it:

```
# graphql-schema-linter --version
module.js:471
    throw err;
    ^

Error: Cannot find module '../../src/rules/input_object_values_have_descriptions'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/src/app/node_modules/graphql-schema-linter/lib/rules/index.js:15:46)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js(module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by cjoudrey** [comment]
🤦‍♂️ well that's embarrassing. I'll fix that typo and I'll double-check why the tests didn't catch that. Thanks.

**Comment by cjoudrey** [comment]
@mscharley this should be fixed in `0.0.21`.

```
$ graphql-schema-linter schema.graphql
schema.graphql
1:4  The object type `Query` is missing a description.
2:12 The field `Query.a` is missing a description.
5:4  The input object type `User` is missing a description.
6:12 The input value `User.username` is missing a description.

✖ 4 errors detected

$ graphql-schema-linter --version
0.0.21
```

Sorry about that and thanks again for reporting the issue. 😀
